delivery_package_fee/tests/test_package_fee.py

```diff
@@ -10,10 +11,15 @@ class TestPackageFee(TransactionCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.product1 = cls.env["product.product"].create(
-            {"name": "Product 1", "type": "product", "lst_price": 1.0}
+            {"name": "Product 1", "is_storable": True, "lst_price": 1.0}
```

Comment:
why?

Reply:
the stock module version 17.0, the `type` field in the `product` model includes an additional value, [product](https://github.com/odoo/odoo/blob/f118a44c5b15163556e3388c5a7daa72119f58f8/addons/stock/models/product.py#L661). However, in version 18.0, this has been replaced by the [is_storable](https://github.com/odoo/odoo/blob/a152910e3a352516ea1240d8833a406b144382dd/addons/stock/models/product.py#L677) field